tests/wikidata/test_extract.py

```diff
@@ -18,12 +18,20 @@
 @pytest.mark.integration
 def test_search_organization_by_label() -> None:
     """Test organization search in wikidata."""
-    expected = "Q26678"
+    expected = "Q679041"
 
-    search_results = list(search_organization_by_label(item_label="BMW"))
+    search_results = search_organization_by_label(item_label='Robert Koch Institute"')
+    if search_results:
+        assert search_results.identifier == expected
+    else:
+        pytest.fail(f"No organizations were found for id: {expected}")
 
-    assert len(search_results) == 2
-    assert search_results[0].identifier == expected
+
+@pytest.mark.integration
+def test_search_organization_by_label_for_none() -> None:
+    """Test organization search in wikidata."""
```

Comment:
```suggestion
```

no need for docstrings in tests, if they don't tell us more than the test name itself (here it has even less informational content)

Reply:
updated in 108989566acf52f31953594e8d8cbd61be22be7a